.editorconfig

```diff
@@ -82,6 +83,7 @@ csharp_prefer_simple_using_statement = true:warning
 csharp_style_prefer_switch_expression = true:suggestion
 dotnet_style_readonly_field = true:suggestion
 dotnet_style_require_accessibility_modifiers = for_non_interface_members:suggestion
+csharp_braces_for_ifelse = required
```

Comment:
I am too lazy to check what each statement does in the editorconfig. Could you add a description in the PR about the changes you are suggesting?